readthedocs/sphinx_domains/admin.py

```diff
@@ -6,10 +6,10 @@
 
 class SphinxDomainAdmin(admin.ModelAdmin):
     list_filter = ('type',)
-    list_display = ('docs_url', 'version', 'build')
-    raw_id_fields = ('project', 'version')
+    list_display = ('__str__', 'version', 'build')
```

Comment:
This uses _way_ fewer queries than doing a `resolve`

Reply:
How did you actually find that out?

Reply:
django-debug-toolbar shows the query count 👍